### DISCUSSION TITLE
convolution performance comparison between cutlass and cudnn

### DISCUSSION THREAD

**Discussion by Peter9606** [NVIDIA/cutlass#209] [Q&A]
It would be great if cutlass team could provide an official performance comparison between cutlass version convolution and cudnn version just like these of gemm between cutlass and cublas, especially for some classic networks such as resnet 50.

**Comment by Peter9606** [comment]
@manishucsd  Is there any performance information public in the [talk](https://gtc21.event.nvidia.com/media/Accelerating%20Convolution%20with%20Tensor%20Cores%20in%20CUTLASS%20%5BS31883%5D/1_ofvh631z) you are going to give in the GTC?

**Reply by manishucsd** [reply]
Hi @Peter9606 , For sure there will be performance results. We will have to wait and find out in the talk what exactly :).   Stay Tuned! See you at the talk
